fsw/src/bp_app.c

```diff
@@ -165,7 +165,7 @@ static CFE_Status_t AppInit(void)
     BP_DoRebuildFlowBitmask();
 
     /* Application startup event message */
-    CFE_EVS_SendEvent(BP_INIT_INF_EID, CFE_EVS_EventType_INFORMATION, "BP App Version %d.%d.%d.%d: Initialized",
+    (void) BPL_EVM_SendEvent(BP_INIT_INF_EID, CFE_EVS_EventType_INFORMATION, "BP App Version %d.%d.%d.%d: Initialized",
                       BP_MAJOR_VERSION, BP_MINOR_VERSION, BP_REVISION, BP_MISSION_REV);
```

Comment:
The previous call to `CFE_EVS_SendEvent` ignored its `CFE_Status_t` return status, so I kept that going here.

However, if we wanted to check the return status, it would look something like this:

```c
    /* Application startup event message */
    BPL_Status_t BPL_EVM_SendEventStatus;
    (void) BPL_EVM_SendEvent(BP_INIT_INF_EID, CFE_EVS_EventType_INFORMATION,
                      "BP App Version %d.%d.%d.%d: Initialized",
                      BP_MAJOR_VERSION, BP_MINOR_VERSION, BP_REVISION, BP_MISSION_REV);
    if (BPL_EVM_SendEventStatus.ReturnValue != BPL_STATUS_SUCCESS)
    {
        fprintf(stderr, "%s(): First BPL_EVM_SendEvent call failed\n", __func__);
        return BPL_EVM_SendEventStatus.ReturnValue;
    }
```

I find this to be a little too distracting for such a benign activity, but I'm happy to go with the group's decision on whether to continue ignoring the status return from Send Event calls.